### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by kamedono**
test pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # raspi-camera-research
-
+test
```
Comment (kamedono) [inline README.md]: what this?

Comment (kamedono) [inline README.md]: why adding?

Comment (kamedono) [inline README.md]: I can't understand....

Comment (kamedonov) [inline README.md]: why adding?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 get header file directory
 > pkg-config --cflags opencv
```
Comment (kamedonov) [inline README.md]: I can't understand

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # raspi-camera-research
```
Comment (kamedonov) [inline README.md]: u r crazy

Comment (kamedonov) [inline README.md]: test comment

**Review by kamedonov — CHANGES_REQUESTED** [review]
fack you

**Pull request closed without merging**